robotbase/src/main/java/frc/robot/commands/elevator/ElevatorAxis.java

```diff
@@ -25,7 +25,5 @@ public boolean isFinished() {
   }
 
   @Override
-  public void end(boolean interrupted) {
-    Robot.elevator.stop();
-  }
+  public void end(boolean interrupted) {}
```

Comment:
Are we sure we want to remove this call to stop the elevator?

Reply:
Yes, removing this causes a stutter when transitioning commands. What should be done is calling `stopMotors` in disabled init. Otherwise we will do the same thing we did in 2023, where the arm tries to go to it's target setpoint after being renabled.